src/errors/contract.ts

```diff
@@ -225,8 +225,8 @@ export class ContractFunctionZeroDataError extends BaseError {
 }
 
 export class RawContractError extends BaseError {
-  code = 3
-  name = 'RawContractError'
+  code = 3 // TODO: What's this code?
```

Comment:
This comes from the node.
```suggestion
  code = 3 
```